src/Okta.Sdk.IntegrationTests/AuthorizationServersScenarios.cs

```diff
@@ -0,0 +1,603 @@
+        {
+            var testClient = TestClient.Create();
+            var defaultAuthorizationServer = await testClient.AuthorizationServers.ListAuthorizationServers().FirstOrDefaultAsync(authorizationServer => (bool)authorizationServer?.Name?.Equals("default"));
+
+            var policies = await testClient.AuthorizationServers.ListAuthorizationServerPolicies(defaultAuthorizationServer.Id).ToListAsync();
+
+            policies.Should().NotBeNull();
+            policies.Count.Should().BeGreaterThan(0);
+        }
+
+        [Fact]
+        public async Task GetAuthorizationServerPolicy()
+        {
+            var testClient = TestClient.Create();
+            var defaultAuthorizationServer = await testClient.AuthorizationServers.ListAuthorizationServers().FirstOrDefaultAsync(authorizationServer => (bool)authorizationServer?.Name?.Equals("default"));
+            var defaultPolicy = await testClient.AuthorizationServers
+                .ListAuthorizationServerPolicies(defaultAuthorizationServer.Id)
+                .FirstOrDefaultAsync(policy => (bool)policy?.Name?.Equals("Default Policy"));
+
+            var retrievedPolicy = await testClient.AuthorizationServers.GetAuthorizationServerPolicyAsync(defaultAuthorizationServer.Id, defaultPolicy.Id);
+            retrievedPolicy.Should().NotBeNull();
+            retrievedPolicy.Id.Should().Be(defaultPolicy.Id);
+            retrievedPolicy.Name.Should().Be(defaultPolicy.Name);
+            retrievedPolicy.Description.Should().Be(defaultPolicy.Description);
+        }
+
+        [Fact(Skip = "Api validation failed: com.saasure.framework.exception.KeyStoreLimitExceededException (400, E0000001): Unable to create new authorization server. You cannot create authorization servers because signing keys could not be generated. To fix this issue, contact support.")]
+        public async Task CreateAuthorizationServerPolicy()
+        {
+            var testClient = TestClient.Create();
+            var testAuthorizationServerName = $"{SdkPrefix}:{nameof(CreateAuthorizationServerPolicy)}:Test Authorization Server ({TestClient.RandomString(6)})";
+
+            var testAuthorizationServer = new AuthorizationServer
+            {
+                Name = testAuthorizationServerName,
+                Description = "Test Authorization Server",
+                Audiences = new string[] { "api://default" },
+            };
+            var testAuthorizationServerPolicy = new OAuthAuthorizationPolicy
+            {
+                Name = $"{SdkPrefix}:{nameof(CreateAuthorizationServerPolicy)}:Test Authorization Policy",
+            };
+            var createdAuthorizationServer = await testClient.AuthorizationServers.CreateAuthorizationServerAsync(testAuthorizationServer);
+            var createdAuthorizationPolicy = await testClient.AuthorizationServers.CreateAuthorizationServerPolicyAsync(testAuthorizationServerPolicy, createdAuthorizationServer.Id);
```

Comment:
Nice! there's a way to create our own policy. We should refactor the other policy tests to assert our own test policy.

Reply:
👍